packages/grafana-toolkit/.eslintrc

```diff
@@ -1,5 +1,14 @@
 {
   "rules": {
     "no-restricted-imports": ["error", { "patterns": ["@grafana/runtime"] }]
-  }
+  },
+  "overrides": [
+    {
+      "files": ["./**/*.{ts,tsx}"],
+      "rules": {
+        "react-hooks/rules-of-hooks": "off",
```

Comment:
Do we have any issue on fixing the code so we can turn these rules on?

Reply:
In grafana toolkit we don't really have react components. Do we? Also it is needed to turn this off because of the useSpinner function that it found the rule.

Reply:
Ah, awesome! I read `grafana-ui`, mybad 🙈